runtime/etc/mli/mli_client_runtime.c

```diff
@@ -74,6 +86,17 @@ void chpl_library_init(int argc, char** argv) {
   // Set up the clientside ZMQ sockets.
   chpl_mli_client_init(&chpl_client);
 
+  chpl_mli_bind(chpl_client.setup_sock, ip, setupPort);
+
+  char* setup_sock_conn = chpl_mli_connection_info(chpl_client.setup_sock);
+  printf("setup socket used %s\n", setup_sock_conn);
+
+  // TODO: pass in argv/argc, connection information
+  spawn_server();
```

Comment:
So I'm not quite clear on the whole namespace situation. I tried to namespace all the functions with `chpl_mli_` (and already realized I forgot some, like `mli_malloc()`, lol), but do we need to do that or not? I know it makes things wordier. I just (perhaps erroneously) didn't want to conflict with any user symbols.

Reply:
Ah, good point.  I did that for the other one and totally forgot here XD